### PR TITLE
Small tweaks post #633/#684

### PR DESCRIPTION
**Pull request by TheBlueMatt**
This fixes one bug that was noticed in review #633 (see https://github.com/rust-bitcoin/rust-lightning/issues/680#issuecomment-689166246 / https://github.com/rust-bitcoin/rust-lightning/pull/633#discussion_r485199875), updates the bindings for both 633 and 684, and tweaks one comment from 633.

### CODE REVIEW

**Review by ariard — COMMENTED** [review]
Code Review ACK 3a025cb

File: lightning/src/ln/chan_utils.rs [hunk 1]
```diff
@@ -349,9 +349,8 @@ pub struct ChannelPublicKeys {
 	/// counterparty to create a secret which the counterparty can reveal to revoke previous
 	/// states.
 	pub revocation_basepoint: PublicKey,
-	/// The public key which receives an immediately spendable primary channel balance in
-	/// a broadcaster's commitment transactions. This key is static across every commitment
-	/// transaction.
+	/// The public key on which the non-broadcaster (ie the countersignatory) receives an immediately
+	/// spendable primary channel balance. This key is static across every commitment transaction.
```
[the remark this answers is not in the document]
Comment (TheBlueMatt) [reply lightning/src/ln/chan_utils.rs]: Its a little wordy now, but at least no one is going to confuse it...better?

**Comment by TheBlueMatt** [comment]
Further updated bindings to include #683.

**Comment by TheBlueMatt** [comment]
Chatted a bit with @ariard about this one on IRC.

**Pull request merged**